.github/styles/config/vocabularies/Industry/accept.txt

```diff
@@ -25,23 +25,33 @@ mintable
 multisig
 namespace
 namespaces
+Nano
+[Nn]ominable
 npm
 npx
 onboarded
 onboarding
+permissionless
 picosecond
 Podman
+[Pp]rometheus
+promtool
 Protobuf
+runbooks
```

Comment:
It's part of the industry vocabulary because it's applicable to development/software engineering. The internet says "Runbooks are a set of standardized written procedures for completing repetitive information technology (IT) processes within a company." 

I came across it in one of the Vale comments against a PR so somebody uses it at least once somewhere hahaha